lib/rules/media-feature-range-notation/README.md

```diff
@@ -84,3 +84,35 @@ The following patterns are _not_ considered problems:
 ```css
 @media (min-width: 1px) and (max-width: 2px) {}
 ```
+
+## Optional secondary options
+
+### `except`
+
+```json
+{ "except": ["array", "of", "options"] }
+```
+
+#### `"exact-value"`
+
+Ignore media features with exact values when the `context` option is selected.
+
+Given:
+
+```json
+{
+  "media-feature-range-notation": ["context", { "except": ["exact-value"] }]
+}
+```
+
```

Comment:
Let's add an example of:

```css
@media (min-width: 1px) {}
```

As a pattern that'll be a problem, as we typically include both for `except` options.

Reply:
Added